### PR TITLE
Add support to get Bitmap snapshots from AirMapView

### PR DESCRIPTION
**Pull request by gpeal**

### CODE REVIEW

File: library/src/main/java/com/airbnb/android/airmapview/AirMapInterface.java [hunk 1]
```diff
@@ -248,4 +247,9 @@
    * Remove GeoJson layer from map, if any.
    */
   void clearGeoJsonLayer();
+
+  /**
+   * Get a Bitmap snapshot of the current
+     */
```
Comment (felipecsl) [inline library/src/main/java/com/airbnb/android/airmapview/AirMapInterface.java]: formatting

Comment (gpeal) [inline library/src/main/java/com/airbnb/android/airmapview/AirMapInterface.java]: done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,20 @@
-# 1.3.3 Mar 28, 2016
+# 1.3.4 April 6, 2016
```
Comment (felipecsl) [inline CHANGELOG.md]: we should only update the changelog after the release has been cut

**Comment by felipecsl** [comment]
:+1: when build finished

**Pull request merged**